src/commands/game_commands/profile.ts

```diff
@@ -59,16 +60,26 @@ export default class ProfileCommand implements BaseCommand {
         {
             example: "`,profile @FortnitePlayer`",
             explanation: "Views FortnitePlayer's player profile.",
+        },
+        {
+            example: "`,profile 141734249702096896`",
+            explanation: "Views a player profile based on their Discord ID.",
         }],
         priority: 50,
     };
 
-    async call({ message }: CommandArgs) {
+    async call({ message, parsedMessage }: CommandArgs) {
         let requestedPlayer: Eris.User;
-        if (message.mentions.length === 1) {
+        if (parsedMessage.components.length === 0) {
+            requestedPlayer = message.author;
+        } else if (message.mentions.length === 1) {
```

Comment:
What happens if a user mentions more than 1 player? Will line 78 give an error or continue silently to the "profile not found" branch?